kotlin/workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/ViewEnvironment.kt

```diff
@@ -17,54 +17,61 @@ package com.squareup.workflow.ui
 
 import kotlin.reflect.KClass
 
+@Suppress("unused")
+@Deprecated(
+    "Renamed to ViewEnvironment.",
+    replaceWith = ReplaceWith("ViewEnvironment", "com.squareup.workflow.ui.ViewEnvironment")
+)
+typealias ContainerHints = ViewEnvironment
```

Comment:
Nice. Should you do the same for the key type?

Reply:
I don't think it's used as ubiquitously (at least an order of magnitude less common to define new keys than to reference the container type), so I think we can just update those references immediately.